plugins/modules/web_infrastructure/jira.py

```diff
@@ -320,12 +371,33 @@ def edit(restbase, user, passwd, params):
     return ret
 
 
+def update(restbase, user, passwd, params):
+    data = {
+        "update": params['fields']
```

Comment:
```suggestion
        "update": params['fields'],
```
good practice for such forms of representing dicts, lists, etc.